<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RssMain</class>
 <widget class="QMainWindow" name="RssMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>681</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>677</width>
          <height>457</height>
         </rect>
        </property>
        <widget class="QTreeWidget" name="resultListWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>681</width>
           <height>461</height>
          </rect>
         </property>
         <column>
          <property name="text">
           <string>标题</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>URL</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>发布日期</string>
          </property>
         </column>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="importBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导入数据库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查询数据库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kwBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>关键字过滤</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionRSS"/>
    <addaction name="actionSys"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="statusTip">
    <string>准备就绪</string>
   </property>
  </widget>
  <action name="actionRSS">
   <property name="text">
    <string>RSS源设置</string>
   </property>
  </action>
  <action name="actionSys">
   <property name="text">
    <string>系统设置</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionKw">
   <property name="text">
    <string>关键字设置</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
